datastore/package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.0.1",
   "private": true,
   "license": "Apache Version 2.0",
+  "bin": {
```

Comment:
authSample follows the convention where the command for executing the sample has the same name as the sample file itself, e.g. `npm run authSample` because the file is named `authSample.js`. So here, the names should be consistent. If you want the command to be `tasks`, then you should rename the file to `tasks.js`